lms/djangoapps/verify_student/models.py

```diff
@@ -919,7 +919,7 @@ def _storage(self):
         if "AWS_SECRET_KEY" in config:
             storage_kwargs["secret_key"] = config["AWS_SECRET_KEY"]
         if "S3_BUCKET" in config:
-            storage_kwargs["bucket"] = config["S3_BUCKET"]
+            storage_kwargs["bucket_name"] = config["S3_BUCKET"]
```

Comment:
Verify_student is using s3boto3 in [prod-config](https://github.com/edx/edx-internal/blob/master/edx-remote-config/prod/lms.yml#L3372).

But I did't find `"S3_BUCKET" in config:` so I am not sure its a valid condition. But args in remote config has bucket_name so that will populate the storage.